### DISCUSSION TITLE
我有个问题想请教

### DISCUSSION THREAD

**Discussion by huhexian** [oldwinter/knowledge-garden#46] [提问] [answered]
贵站（garden.oldwinter.top）应该是使用quartz方案建成的吧，我看了一下quartz的文档，似乎是基于hugo，是不是我先得在本地用hugo生成一个网站，再使用quartz呢？

**Comment by oldwinter** [comment, marked as answer]
可以直接看quaetz官方文档哦。我觉得算是hugo的一个主题，只不过这个主题的定制成分很大。本地可建可不见，更方便调试而已。我由于比较熟练，我是直接创建部署到云上了的。
